tests/unit/test_cluster.py

```diff
@@ -273,7 +273,7 @@ def test_render_patroni_yml_file(self, _, _render_file, _get_postgresql_version)
         _render_file.assert_called_once_with(
             "/var/snap/charmed-postgresql/current/etc/patroni/patroni.yaml",
             expected_content,
-            0o644,
+            0o600,
```

Comment:
Should we keep the group permissions here?

Reply:
From what I remember, we don't need that.